src/ARCtrl/WebRequest/WebRequest.fs

```diff
@@ -0,0 +1,56 @@
+﻿module ARCtrl.WebRequest
+
+open Fable.Core
+open Fable.SimpleHttp
+open Fable.Core.JsInterop
+
+open Fable.SimpleHttp 
+
+[<RequireQualifiedAccess>]
+module MyNodeJs =
```

Comment:
module name?

Reply:
Or could you shortly explain what this module is for?

Reply:
This module contains js fable logic which only applied in a nodejs environment. 

This complete file should be restructured, but i am not sure what the best approach is. 

Maybe i should add a "Web" type containing requests as static members.

Reply:
Maybe just cut the `My`. Already `NodeJS.stuff` would feel a bit more concise.